website/addons/owncloud/routes.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from framework.routing import Rule, json_renderer
+
+from . import views
```

Comment:
Prefer absolute over relative module import paths, but this isn't a blocker. (As stated elsewhere)